doc/circulationrules.md

```diff
@@ -308,3 +328,43 @@ For `priority: first-line` it must be after the last rule.
 For one tenant there is only a single circulation rules file.
 
 It must have a single priority line and a single fallback-policy line.
+
+## Loan Rules UX Design Video
+
+The video from May 2017 explains the user experience (UX) design of the loan rules.
+The general principles still apply but note that some aspects have changed since then
+or haven't been implemented yet.
```

Comment:
Given this document describes it's purpose as describing what has been implemented, why is it valuable to include a video with a comment that refers to it being inconsistent with what has been implemented?

Reply:
Because "The general principles still apply".
It is valuable to link requirements that haven't been implemented yet but may get implemented in future.
I've got a dozen requests to give a talk about the circulation rules document during the last three years and I could point to the video and it was useful.
If you want the video been changed to adopt the latest mod-circulation changes please create an issue so that it can get prioritized.

Reply:
I personally don't think that requirements that are not implemented should be described in documentation for an implementation. I think future requirements are better described in the requirements space, e.g. wiki.

I will resolve this conversation and approve this request because it's likely not valuable enough to debate this